app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/AddEvent.kt

```diff
@@ -48,8 +48,8 @@ class AddEvent : Activity(), AdapterView.OnItemSelectedListener {
 
             // converts the entered time to text and prints
             val time = findViewById<EditText>(R.id.editTextTime).text.toString()
-            System.out.println(time)
-            System.out.println(timeSpecification)
+            println(time)
+             println(timeSpecification)
```

Comment:
Instead of shortening the syntax, better option:
Use proper logging instead of println().
- Logging has the concept of priority (error, debug, warn, etc.), which println doesn't have.
- No one will see println(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with println.